app/assets/javascripts/users/settings/organization.js

```diff
@@ -236,8 +235,19 @@ function initRemoveUsers() {
   );
 }
 
+function initReloadPageAfterInviteUsers() {
+  $('[data-id=org-invite-users-modal]')
+  .on('hidden.bs.modal', function() {
+    if ($(this).attr('data-invited') !== undefined) {
```

Comment:
here you can use _.isUndefined() method from undersorejs 

http://underscorejs.org/#isUndefined